website/src/app/kb/user-guides/linux-client/readme.mdx

```diff
@@ -1,88 +1,85 @@
 
+Mark the binary as executable:
+
+```bash
+chmod +x firezone-client-headless-linux_1.0.0_x86_64
+```
+
 ## Usage
 
-The Linux Client requires a Service Account token to authenticate to Firezone.
-If you don't already have a token generated, follow the instructions in the
-[Service Account](/kb/authenticate/service-accounts) documentation.
+### Signing in
 
-Once you have a token, you can start the Linux Client using the following
-command:
+Headless Clients require a service account token to authenticate to Firezone.
+You can generate a token using the instructions in the
+[service account](/kb/authenticate/service-accounts) documentation.
 
+Once you have a token, you can start the Client using the following command:
+
+```bash
+sudo FIREZONE_TOKEN=<TOKEN> ./firezone-client-headless-linux_1.0.0_x86_64
 ```
-sudo FIREZONE_TOKEN=<TOKEN> ./headless-client-x64
-```
+
+<Alert color="info">
+  `sudo` is required to control the system's DNS.
+</Alert>
 
 Set some environment variables to configure it:
 
-```
-export FIREZONE_NAME="Development Webserver"
-export FIREZONE_ID="some unique identifier"
+```bash
+export FIREZONE_NAME="Development API test client"
+export FIREZONE_ID=$(uuidgen)
 export FIREZONE_TOKEN=<TOKEN>
-export DNS_CONTROL="systemd-resolved" # or "etc-resolv-conf"
+export FIREZONE_DNS_CONTROL="systemd-resolved" # or "etc-resolv-conf"
 export LOG_DIR="./"
-sudo -E ./headless-client-x64
+sudo -E ./firezone-client-headless-linux_1.0.0_x86_64
 ```
 
 See [below](#environment-variable-reference) for a full list of environment
 variables.
 
-A sample output of the help command is shown below:
-
-```
-> sudo ./headless-client-x64 -h
+### Accessing a Resource
 
-Usage: headless-client [OPTIONS] [COMMAND]
+When Firezone is signed in, HTTP clients, SQL clients, and other programs will automatically use it to securely connect to Resources.
 
-Commands:
-  standalone  Act as a CLI-only Client
-  help        Print this message or the help of the given subcommand(s)
 
-Options:
-      --token-path <TOKEN_PATH>
-          A filesystem path where the token can be found [env: FIREZONE_TOKEN_PATH=] [default: /etc/dev.firezone.client/token]
-  -i, --firezone-id <FIREZONE_ID>
-          Identifier used by the portal to identify and display the device [env: FIREZONE_ID=]
-  -l, --log-dir <LOG_DIR>
-          File logging directory. Should be a path that's writeable by the current user [env: LOG_DIR=]
-  -m, --max-partition-time <MAX_PARTITION_TIME>
-          Maximum length of time to retry connecting to the portal if we're having internet issues or it's down. Accepts human times. e.g. "5m" or "1h" or "30d" [env: MAX_PARTITION_TIME=]
-  -h, --help
-          Print help
-  -V, --version
-          Print version
-```
 
 ### Split DNS
 
-By default, Split DNS is **disabled** for the Linux Client. This means that
-access to DNS-based Resources won't be routed through Firezone.
+By default, Split DNS is **disabled** for the Linux headless Client. This means
```

Comment:
I still worry about this UX as I think it's going to surprise nearly everyone that runs the headless client. Maybe the next thing to work on.

Reply:
I'd be okay with bailing out if the user doesn't specify any DNS control. Alternately we could make systemd-resolved the default, but then that default becomes part of the CLI interface forever, not sure if I like that.